### PR TITLE
Fix UG typos

### PR DESCRIPTION
**Pull request by ldinghan**
Many typos relating to punctuation, standardisation and mistyped words.

Fixing these typos will make the UG more ready for grading by those who intentionally seek errors in language, typos, etc.

Also re-ordered some of the titles and command summary to better fit in their categories.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1400,6 +1401,7 @@ Format: `createsession s/SESSION_NUMBER n/STUDENT_NAME…`
 :information_source: **Note**:
 
 * The name of the student must exist in F.A.K.E.J.A.R.V.I.S.
+* The session with the session number must not already exist in F.A.K.E.J.A.R.V.I.S.
```
Comment (Respirayson) [inline docs/UserGuide.md]: must not already exist sounds weird to me

perhaps can be like must **not** exist

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1638,10 +1640,10 @@ You can remove your student(s) from a consultation if you need to cancel your st
...
 Format: `removefromconsult CONSULTATION_INDEX n/STUDENT_NAME…`
 
-|   **Parameter**    |                       **Description**                        |
-|:------------------:|:------------------------------------------------------------:|
-| CONSULTATION_INDEX | The index of the consultation in the upcoming consultations. |
-|    STUDENT_NAME    |   The name of the student to be added to the consultation.   |
+|   **Parameter**    |                           **Description**                            |
+|:------------------:|:--------------------------------------------------------------------:|
+| CONSULTATION_INDEX | The index of the consultation in the list of upcoming consultations. |
+|    STUDENT_NAME    |     The name of the student to be removed from the consultation.     |
```
Comment (Respirayson) [inline docs/UserGuide.md:1646]: good catch oops

**Review by Respirayson — COMMENTED** [review]
LGTM, but one small nit to look at

File: docs/UserGuide.md (hunk 1 above)
Comment (ldinghan) [reply docs/UserGuide.md]: Changed it!

**Review by raydenlim — APPROVED** [review]
Nothing jarring. LGTM

**Pull request merged**